CONTRIBUTING.md

```diff
@@ -24,14 +24,13 @@ and feel free to propose changes to this document in a pull request.
 
 ## Code of Conduct
 * This project is governed by the [permissioning-smart-contracts Code of Conduct](CODE_OF_CONDUCT.md). By participating, 
-you are agreeing to uphold this code. Please report unacceptable behavior. Please report unacceptable behavior to [private@pegasys.tech].
+you are agreeing to uphold this code. Please report unacceptable behavior. Please report unacceptable behavior to [private-quorum@consensys.net].
 
 ## I just have a quick question
 
 > **Note:** Please don't file an issue to ask a question.  You'll get faster results by using the resources below.
 
-* [permissioning-smart-contracts documentation]
-* [Gitter]
+* [RocketChat]
```

Comment:
These arent part of the HLF, should comms be directed to the ConsenSys Discord instead?